packages/foreman/rubygem-rsec/rubygem-rsec.spec

```
@@ -1,69 +1,88 @@
-# Generated from rsec-0.4.2.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name rsec
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.4.3
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Extreme Fast Parser Combinator for Ruby
 Group: Development/Languages
-License: Ruby or BSD 
+License: FIXME
```

I guess this is unchanged

```suggestion
License: Ruby or BSD
```